panelManager.js

```diff
@@ -350,7 +352,7 @@ var dtpPanelManager = Utils.defineClass({
         LookingGlass.LookingGlass.prototype.open = LookingGlass.LookingGlass.prototype._oldOpen;
         delete LookingGlass.LookingGlass.prototype._oldOpen
 
-        delete Main.panel.style;
+        Object.defineProperty(Main.panel, "style", { value: this._oldMainPanelStyle, writable: true });
```

Comment:
I think this might return a static string value from whatever Main.panel.style was before enable, rather than restore original functionality?